lib/modules/modules.go

```diff
@@ -38,184 +38,130 @@ import (
-	// enterprise editions (team, cloud, on-prem). Historically, access request
-	// was only available for enterprise cloud and enterprise on-prem.
-	AdvancedAccessWorkflows bool
+	// --------------- Cloud Settings
 	// Cloud enables some cloud-related features
 	Cloud bool
-	// HSM enables PKCS#11 HSM support
-	HSM bool
-	// Desktop enables desktop access product
-	Desktop bool
+	// CustomTheme holds the name of WebUI custom theme.
+	CustomTheme string
+	// IsStripeManaged indicates if the cluster billing is managed via Stripe
+	IsStripeManaged bool
+	// IsUsageBasedBilling enables some usage-based billing features
+	IsUsageBasedBilling bool
+	// Questionnaire indicates whether cluster users should get an onboarding questionnaire
+	Questionnaire bool
+	// SupportType indicates the type of customer's support
+	SupportType proto.SupportType
+	// Entitlements reflect Cloud Entitlements including access and limits
+	Entitlements map[entitlements.EntitlementKind]EntitlementInfo
+
+	// todo (michellescripts) have the following fields evaluated for deprecation, consolidation, or fetch from Cloud
+	// AdvancedAccessWorkflows is currently set to the value of the Cloud Access Requests entitlement
+	AdvancedAccessWorkflows bool
 	// RecoveryCodes enables account recovery codes
 	RecoveryCodes bool
 	// Plugins enables hosted plugins
 	Plugins bool
 	// AutomaticUpgrades enables automatic upgrades of agents/services.
 	AutomaticUpgrades bool
-	// IsUsageBasedBilling enables some usage-based billing features
-	IsUsageBasedBilling bool
-	// DeviceTrust holds its namesake feature settings.
-	DeviceTrust DeviceTrustFeature
-	// FeatureHiding enables hiding features from being discoverable for users who don't have the necessary permissions.
-	FeatureHiding bool
-	// AccessRequests holds its namesake feature settings.
-	AccessRequests AccessRequestsFeature
-	// CustomTheme holds the name of WebUI custom theme.
-	CustomTheme string
-
 	// AccessGraph enables the usage of access graph.
 	// NOTE: this is a legacy flag that is currently used to signal
 	// that Access Graph integration is *enabled* on a cluster.
 	// *Access* to the feature is gated on the `Policy` flag.
 	// TODO(justinas): remove this field once "TAG enabled" status is moved to a resource in the backend.
 	AccessGraph bool
-	// IdentityGovernanceSecurity indicates whether IGS related features are enabled:
-	// access list, access request, access monitoring, device trust.
-	IdentityGovernanceSecurity bool
-	// AccessList holds its namesake feature settings.
-	AccessList AccessListFeature
-	// AccessMonitoring holds its namesake feature settings.
-	AccessMonitoring AccessMonitoringFeature
+	// AccessMonitoringFeature enables the usage of access monitoring.
+	// NOTE: this flag is used to signal that Access Monitoring is *enabled* on a cluster.
+	// *Access* to the feature is gated on the `AccessMonitoring` entitlement.
+	AccessMonitoringFeature bool
```

Comment:
should we just rename this to `AccessMonitoringConfigured`? b/c i think `AccessMonitoringFeature` sounds too much like `is cluster entitled to this feature` (up to you)